src/Calibration/Detector.cpp

```diff
@@ -12,6 +12,7 @@ namespace ZividPython
     void wrapClass(pybind11::class_<Zivid::Calibration::DetectionResult> pyClass)
     {
         pyClass.def("valid", &Zivid::Calibration::DetectionResult::valid)
+            .def("__bool__", &Zivid::Calibration::DetectionResult::operator bool)
```

Comment:
This is fine, but even better is to use `constexper` and `std::is_detected` and add `"__bool__"` to all wrapped classes automatically when supported.

You can do that in https://github.com/zivid/zivid-python/blob/master/src/include/ZividPython/Wrappers.h#L75 and then this is fixed once and for all.

Reply:
Good idea. I'll give it a shot.

Reply:
I just pushes a fixup that does this.

Problem: The compiler on Windows doesn't have `<experimental/type_traits>`.
Possible solution: Just copy in the "possible implementation" from https://en.cppreference.com/w/cpp/experimental/is_detected ? 

What do you think?

Reply:
I think that is fine. I would add it to a separate header and use `#if __has_include` so the unofficial implementation is only used when needed.

Reply:
Added a new solution now. Please have a close-ish look, because I'm on super thin ice here, C++-wise...

Also temporarily added a unit test that used to fail and now succeeds. It adds about 2 minutes to the CI (+14%).